mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 site_name: Charmed Magma
-repo_url: https://github.com/canonical/charmed-magma
+repo_url: https://github.com/canonical/charmed-magma/latest
```

Comment:
Is this a real URL? I'm aware the PR is already merged

Reply:
Yes the url should work, it did locally 
once you issue `mike deploy`.

Reply:
Ok, I get a 404 error when copying this URL in my browser

Reply:
The github action will fix it, soon.
it will run the correct `mike deploy`